graphene_django/fields.py

```diff
@@ -147,21 +147,19 @@ def resolve_connection(cls, connection, args, iterable, max_limit=None):
 
         if isinstance(iterable, QuerySet):
             list_length = iterable.count()
-            list_slice_length = (
-                min(max_limit, list_length) if max_limit is not None else list_length
-            )
         else:
             list_length = len(iterable)
-            list_slice_length = (
-                min(max_limit, list_length) if max_limit is not None else list_length
-            )
+
+        list_slice_length = list_length
+        if max_limit is not None and "last" not in args:
```

Comment:
Without checking for `last` here and below, `list_slice_length` was being set to 4, and `after` to `0`, slicing off reporters 5-8.

Query for last 2 reporters should be getting 7 and 8, but it wasn't getting those correctly.